java/test/apps/startup/StartupActionsPreferencesPanelTest.java

```diff
@@ -1,32 +1,52 @@
 package apps.startup;
 
+import jmri.swing.PreferencesPanelTestBase;
 import jmri.util.JUnitUtil;
 
 import org.junit.jupiter.api.*;
 import org.junit.Assert;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 /**
- *
  * @author Paul Bender Copyright (C) 2017
  */
-public class StartupActionsPreferencesPanelTest {
+public class StartupActionsPreferencesPanelTest extends PreferencesPanelTestBase {
+
+    @Override
+    @BeforeEach
+    public void setUp() {
+        JUnitUtil.setUp();
+        jmri.util.JUnitUtil.initStartupActionsManager();
+        prefsPanel = new StartupActionsPreferencesPanel();
+    }
 
+    @Override
     @Test
-    public void testCTor() {
-        StartupActionsPreferencesPanel t = new StartupActionsPreferencesPanel();
-        Assert.assertNotNull("exists",t);
+    public void getLabelKey(){
+        // This class returns null for label key, but should it?
+        assertThat(prefsPanel.getLabelKey()).isNull();
     }
 
-    @BeforeEach
-    public void setUp() {
-        JUnitUtil.setUp();        jmri.util.JUnitUtil.initStartupActionsManager();
+    @Override
+    @Test
+    public void getPreferencesTooltip(){
+        // This class returns null for preferences tool tip, but should it?
+        assertThat(prefsPanel.getPreferencesTooltip()).isNull();
     }
 
-    @AfterEach
-    public void tearDown() {
-        JUnitUtil.tearDown();
+    @Override
+    @Test
+    public void getTabbedPreferencesTitle(){
```

Comment:
Since a preferences panel controls if it is put into a tab with another preference panel or not, you should expect that preferences panels that are not under tabs will return null for the title for the tab.

Reply:
thanks.  will fix.